### DISCUSSION TITLE
Simpler parameters to generate DCA airfoil

### DISCUSSION THREAD

**Discussion by afshawnlotfi** [NAnand-TUD/parablade#9] [Q&A] [answered]
Hi right now I am using this formula to calculate a DCA airfoil (https://afshawn.org/jetengine/docs/Circular%20Arc%20Airfoil.html) with only the parameters 
<img width="275" alt="image" src="https://user-images.githubusercontent.com/6283745/166122996-fb01a2fb-1808-4e08-868c-c564c5852ea8.png">

Is this possible to reproduce as easily with Parablade? I was looking at the example here and it seems you need a lot more parameters like dist[1-4], etc.
https://github.com/NAnand-TUD/parablade/blob/master/examples/blade_2D_connecting_arcs_demo.py

**Comment by NAnand-TUD** [comment, marked as answer]
Hello ! 

In ParaBlade we define multiple thicknesses to control the surface. The kind of parameterization that you want cannot be done with the current version of ParaBlade. 

Thanks for dropping your query.
Nitish
